frontend/src/modules/EditZing/Components/GroupGrid.tsx

```diff
@@ -1,54 +1,68 @@
-import React from 'react'
+import { useState } from 'react'
 import { StudentGrid } from 'EditZing/Components/StudentGrid'
-import Grid from '@mui/material/Grid'
+import Grid, { GridSize } from '@mui/material/Grid'
 import { GroupGridProps } from 'EditZing/Types/ComponentProps'
 import { useDrop } from 'react-dnd'
 import { STUDENT_TYPE, DnDStudentTransferType } from 'EditZing/Types/Student'
 import {
   StyledGroupText,
   StyledGroupTextWrapper,
   StyledGroupContainer,
-  StyledMetricBox,
-  StyledMetricText,
+  // StyledMetricBox,
+  // StyledMetricText,
 } from 'EditZing/Styles/StudentAndGroup.style'
 
-/** the equivalent of Column */
 export const GroupGrid = ({
   studentList,
   groupIndex,
   moveStudentBetweenGrids,
   moveStudentWithinGrid,
+  zingId,
```

Comment:
FYI, since we deprecated the use of the word "Zing" to represent ids the variable `zingId` has now been renamed to `courseId` in #55, which has since been merged to main